site/src/scss/main.scss

```diff
@@ -4,7 +4,10 @@
 
 @include foundation-everything();
 
+@import "../fonts/fontello/css/fontello.css";
```

Comment:
It looks like fontello is being imported twice - once in scss/_font.scss, and once here.  I think we can remove scss/_font.scss?

Reply:
Deleted!